ebos/eclcpgridvanguard.hh

```diff
@@ -351,16 +352,35 @@ protected:
             input_grid = &this->eclState().getInputGrid();
             porv = this->eclState().fieldProps().porv(true);
             aquifer_cell_volumes = this->eclState().aquifer().numericalAquifers().aquiferCellVolumes();
+            OpmLog::info("\nProcessing grid");
         }
 
         grid_.reset(new Dune::CpGrid());
-        grid_->processEclipseFormat(input_grid,
-                                    /*isPeriodic=*/false,
-                                    /*flipNormals=*/false,
-                                    /*clipZ=*/false,
-                                    porv,
-                                    this->eclState().getInputNNC(),
-                                    aquifer_cell_volumes);
+        const auto& removed_cells = grid_->processEclipseFormat(input_grid,
+                                                                /*isPeriodic=*/false,
+                                                                /*flipNormals=*/false,
+                                                                /*clipZ=*/false,
+                                                                porv,
+                                                                this->eclState().getInputNNC(),
+                                                                aquifer_cell_volumes);
+
+        if (mpiRank == 0) {
+            const auto& unit_system = this->eclState().getUnits();
+            const auto& volume_unit = unit_system.name( UnitSystem::measure::geometric_volume);
+            const auto& active_volume = input_grid->activeVolume();
+            double total_volume = unit_system.from_si( UnitSystem::measure::geometric_volume, std::accumulate(active_volume.begin(), active_volume.end(), 0.0));
+            OpmLog::info(fmt::format("Total number of active cells: {} / total volume: {:0.0f} {}", grid_->numCells(), total_volume , volume_unit));
+
+            double removed_volume = 0;
+            for (const auto& cell_index : removed_cells)
+                removed_volume += input_grid->getCellVolume(cell_index);
+
+            if (removed_volume > 0) {
+                removed_volume = unit_system.from_si( UnitSystem::measure::geometric_volume, removed_volume );
+                OpmLog::info(fmt::format("Removed {} cells with a total volume of {:0.0f} {} ({:5.3f} %) due to PINCH", removed_cells.size(), removed_volume, volume_unit, 100 * removed_volume / total_volume));
+            }
```

Comment:
Any chance we report the deactivated pore-volume too?

Reply:
Sure; I'll leave this open for @alfbr to comment - but we can report any combination geometric-volume and/or pore-volume

Reply:
Actually, I am a bit confused, so I think I at least need to understand better before making any comment. Is it not MINPV (or MINPVV) that actually removes the pore volume?

Reply:
Yes - you are right. Have upated.